dialog/base.go

```diff
@@ -40,7 +40,7 @@ type dialog struct {
 	desiredSize fyne.Size
 
 	win            *widget.PopUp
-	bg             *canvas.Rectangle
+	bg             *themedBackground
```

Comment:
@andydotxyz I finally realized why `dialogTitle` was important (although it was clearly written in comments what it does, I didn't understand before). `dialogTitle` had the mission to update the background based on the theme. So, should I keep `dialogTitle` or is it better to have a widget called `themedBackground` as the one added here?

Personally, I don't like `dialogTitle` because it is a bit confusing (it is a child that updates its parent), although clearly it is the shortest solution.

Reply:
Now that `BackgroundColor` is removed from `WidgetRenderer` is this `bg` needed at all?

Reply:
I would say it is still needed, because that background is used to add transparency to the 'background' icon:

With background:
<img width="166" alt="Captura de Pantalla 2021-02-18 a la(s) 16 08 19" src="https://user-images.githubusercontent.com/12239342/108421790-adb94280-7203-11eb-9407-d0b1d9433baa.png">

Without background:
<img width="172" alt="Captura de Pantalla 2021-02-18 a la(s) 16 07 14" src="https://user-images.githubusercontent.com/12239342/108421853-c1fd3f80-7203-11eb-9b54-38e6715b9883.png">

Reply:
Couldn’t we change the background image colour instead?

Reply:
Yes, that is a possibility, however it would mean to have new functions `xyzThemedResource` to deal with this new color 🤔

Reply:
Unfortunately we can’t - the renderer does not currently support a faded image with an alpha channel. This BG was a workaround for that, I remember now

Reply:
I see, then I will resolve this conversation.